.circleci/config.yml

```diff
@@ -21,14 +21,29 @@ jobs:
       - run:
           name: Install Dependencies
           command: yarn
+      - run:
+          name: Build all code to JavaScript
+          command: yarn build
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - run:
+          name: Create CLI binaries
+          command: npx pkg --out-path ./cli-binaries ./packages/cli/
       - save_cache:
           name: Save Yarn Package Cache
           key: yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
           paths:
             - node_modules/
+      - run:
+          command: SPEC=petstore.oas2.json BINARY=prism-cli-linux yarn run.binary
+          background: true
+      - run:
+          shell: /bin/sh
+          command: |
+            wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 10 http://localhost:4010
+            :
       - run:
           name: Run tests
-          command: yarn test.coverage --verbose
```

Comment:
We can do both :D

Reply:
Sorry what I meant was we should have normal tests and binary tests as two different steps because they likely won’t be used together forever.

Reply:
does `likely` satisfy investing time now to separate these tests?

Reply:
It’s more that we should not be confusing these two very different test suites. We will likely separate them in Circle very soon but we will definitely be running them separately locally. 

yarn test is for source code which devs will run all the time 
yarn test.binary is for the built product which developers will run rarely (before submitting PR or a release maybe, or maybe not) 

It’s better to keep separate test suites seperate, so we can reference them however we want on circle.